### PR TITLE
Added RT4 item definition from cache

### PR DESCRIPTION
**Pull request by Logicail**
Can you check Item.name()

RT6 can use Component#itemName() but that doesn't exist for RT4.
I'm not sure if using itemConfig will always work or if it should just always load from the cache.

Tested the class loading items 0-13352

### CODE REVIEW

File: rt4/Item.java [hunk 1]
```diff
@@ -53,7 +54,14 @@ public Point centerPoint() {
 
 	@Override
 	public String name() {
-		return StringUtils.stripHtml(ItemConfig.getDef(ctx, id).getName());
+		final String name;
+		if (component != null && component.itemId() == id) {
+			name = ItemConfig.getDef(ctx, id).getName();
```
Comment (Logicail) [inline rt4/Item.java]: Not sure about this RT6 uses Component#itemName()

File: rt4/CacheItemConfig.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                    modelOffsetY -= 65536;
+                }
+            } else if (opcode == 11) {
+                stackable = true;
+            } else if (opcode == 12) {
+                value = stream.getInt();
+            } else if (opcode == 16) {
+                members = true;
+            } else if (opcode == 23) {
+                final int i = stream.getUInt24();
```
Comment (Logicail) [inline rt4/CacheItemConfig.java]: 23 & 25 actually read ushort and ubyte

**Comment by Logicail** [comment]
#1110

File: rt4/CacheItemConfig.java (hunk 2 above)
Comment (Timer) [inline rt4/CacheItemConfig.java]: respectively or both in each?
you changing this or want me to go through it all?

Comment (Logicail) [inline rt4/CacheItemConfig.java]: Both in each, the value isn't used anyway and reading 2+1 byte is the same as 3 bytes.

**Pull request merged**